EE page styling issue fix

shawn320 opened the pull request:
# Description

- Fixed an styling issue pointed out by Nelia. The project phase text should be aligned at the bottom for French content (used to allign at center). 
- Fixed splash page button to center the button text

![Capture](https://user-images.githubusercontent.com/64853947/223776560-6f44661c-f038-4023-b225-b89c07714528.PNG)

will0684 commented:
Looks like this branch needs to merge with main to get caught up to the new changes with the `<Link>` elements. I did that on my end and the placement of the content looks as intended. Only thing I would note is the project status tooltip looks lower than it was before when it comes up. You can see that here:

**main**

<img width="644" alt="Screenshot 2023-03-08 at 12 08 51 PM" src="https://user-images.githubusercontent.com/31868510/223836978-d34aa3d6-fd3e-44c5-97e3-ffd1302ad749.png">

**fix-projectinfo-style**

<img width="642" alt="Screenshot 2023-03-08 at 12 06 25 PM" src="https://user-images.githubusercontent.com/31868510/223836100-fd712e81-4c13-495c-8a37-42370aecae4c.png">

shawn320 commented:
> Looks like this branch needs to merge with main to get caught up to the new changes with the `<Link>` elements. I did that on my end and the placement of the content looks as intended. Only thing I would note is the project status tooltip looks lower than it was before when it comes up. You can see that here:
> 
> **main**
> 
> <img alt="Screenshot 2023-03-08 at 12 08 51 PM" width="644" src="https://user-images.githubusercontent.com/31868510/223836978-d34aa3d6-fd3e-44c5-97e3-ffd1302ad749.png">
> 
> **fix-projectinfo-style**
> 
> <img alt="Screenshot 2023-03-08 at 12 06 25 PM" width="642" src="https://user-images.githubusercontent.com/31868510/223836100-fd712e81-4c13-495c-8a37-42370aecae4c.png">

Thank you, I completely missed the tooltip, updated. Also updated with the main branch.

will0684 reviewed (approved):
Thank you!
